src/universal/modules/welcome/components/ProgressDots/ProgressDots.js

```diff
@@ -79,25 +85,24 @@ styles = StyleSheet.create({
     backgroundColor: '#fff',
     border: `2px solid ${theme.palette.mid50l}`,
     borderRadius: '100%',
-    // NOTE: Removing cursor until dots can change navigation
     cursor: 'default',
     display: 'inline-block',
     height: '1rem',
     margin: '0 .375rem',
     width: '1rem'
   },
 
-  // NOTE: Same thing, different semantics (completed, current)
   progressDotCompleted: {
-    backgroundColor: theme.palette.mid50l
+    backgroundColor: theme.palette.mid50l,
+    cursor: 'pointer'
   },
+
   progressDotCurrent: {
     backgroundColor: theme.palette.warm,
     borderColor: theme.palette.warm
   },
 
   progressDotLabel: {
-    // TODO: Make mixin for Sass: @include sr-only;
     border: 0,
     clip: 'rect(0, 0, 0, 0)',
     height: '1px',
```

Comment:
I see we have 2 `ProgressDots`. 1 for welcome & 1 for meeting. we should combine those in another PR.